gammapy/utils/coordinates/other.py

```diff
@@ -73,12 +73,12 @@ def velocity_glon_glat(x, y, z, vx, vy, vz):
 
 def motion_since_birth(v, age, theta, phi):
     """
-    Compute motion of a object with given velocity, direction and age.
+    Compute motion of an object with given velocity, direction and age.
```

Comment:
`object` kind of clashes with python `object`. We could use `source` or `astrophysics object`?